dags/openshift_nightlies/util/var_loader.py

```diff
@@ -49,7 +49,7 @@ def get_config_vars(release: OpenshiftRelease, task="install", config_dir=f"{con
 
 def get_default_task_vars(release: OpenshiftRelease, task="install", task_dir=f"{constants.root_dag_dir}/tasks"):
     if task == "install":
-        if release.platform == "aws" or release.platform == "azure" or release.platform == "gcp" or release.platform == "alibaba":
+        if release.platform == "aws" or release.platform == "aws-arm" or release.platform == "azure" or release.platform == "gcp" or release.platform == "alibaba":
```

Comment:
if release.platform in ("aws!", "aws-arm", .....):
we can use this pattern as we are adding more options

Reply:
Good call. Updated the condition, added few more tests, and refactored existing tests to be clearer (I hope).